CHANGELOG.md

```diff
@@ -99,10 +99,11 @@ For status on this, see https://github.com/googleapis/google-cloud-cpp/issues/88
 
 ### New Libraries
 
-We are introducing a new client library for [Cloud Run]. While we do not
-anticipate any API changes to this library before declaring it GA, we are
-releasing it early in case it elicits some feedback that requires changes.
+We are introducing new client libraries for [Cloud Run]. While we do not
+anticipate any API changes to these libraries before declaring them GA, we are
+releasing them early in case they elicit some feedback that requires changes.
 
+[Cloud Bh]: https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/batch/README.md
 [Cloud Run]: https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/run/README.md
```

Comment:
I would s/ this whole thing with:

```
We are introducing 2 new client libraries for GCP services. While we do not
anticipate any API changes to these libraries before declaring them GA, we are
releasing them early in case they elicit some feedback that requires changes.

* [Batch](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/batch/README.md)
* [Cloud Run](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/run/README.md)
```

Reply:
Done